app/code/Magento/Backend/Block/Dashboard/Orders/Grid.php

```diff
@@ -5,36 +5,42 @@
  */
 namespace Magento\Backend\Block\Dashboard\Orders;
 
+use Magento\Backend\Block\Template\Context;
+use Magento\Backend\Helper\Data;
+use Magento\Framework\Module\Manager;
+use Magento\Reports\Model\ResourceModel\Order\CollectionFactory;
+
 /**
  * Adminhtml dashboard recent orders grid
  *
+ * @api
```

Comment:
@ihor-sviziev after I moved adding dashboard blocks fro `_prepareLayout` method to xml files, I got failure in (i think) static tests that I added block that is not marked as api to layout. So I was sure this is requirement in that case. What should I do with this?

Reply:
Could you point me to test result?

Reply:
I faced this when this PR was draft, I force pushed changes later. I can remove one annotation and push the changes for testing purpose

Reply:
@ihor-sviziev here it comes https://public-results-storage-prod.magento-testing-service.engineering/reports/magento/magento2/pull/26923/3019ba1dfb621066ed1bd93a177981b8/Statics/allure-report-b2b/index.html#categories/9f143fbe76131fd0113101cfa4fa3008/85c8b92bfa927797/

Reply:
![image](https://user-images.githubusercontent.com/9566840/75261486-6ca67200-57eb-11ea-86bf-7a1a179e7b0f.png)
Posting image also since this report might vanish later when I'll force push reverting last commit

Reply:
@Bartlomiejsz we discussed it, blocks should be marked as `@api`, you did everything correct last time. That was something new for me, sorry for confusing